terraform/modules/self-service/modules/cognito/cognito.tf

```diff
@@ -104,24 +104,24 @@ resource "aws_cognito_user_pool" "user_pool" {
     invite_message_template {
       email_subject = "You have been invited to collaborate on the GOV.UK Verify Manage certificates service"
       email_message = <<-EOT
-        Dear {username}
-
-        You have been invited to collaborate on the GOV.UK Verify Manage certificates service.
-
-        Sign in at ${var.domain} using the following temporary password:
-
-        {####}
-
-        You will be asked to create a new password and set up multi-factor authentication using your preferred authentication app.
-
-        Please sign in within 24 hours, otherwise the temporary password will expire.
-
-        If you miss this deadline, contact your admin to ask for another temporary password.
-
-        Thanks
-
-        The GOV.UK Verify team
-        https://www.verify.service.gov.uk/
+        <p>Dear {username}</p>
+        
+        <p>You have been invited to collaborate on the GOV.UK Verify Manage certificates service.</p>
```

Comment:
The capitalisation here is a bit wacky. If we are capitalising the name of the service, then the C and the S should be capitals too. If not, then the M should not be capitalised.